File: PathoFroh/src/main/java/com/patho/main/repository/jpa/custom/ListItemRepositoryCustom.java
package com.patho.main.repository.jpa.custom;


import com.patho.main.model.preset.ListItem;
import com.patho.main.model.preset.ListItemType;

import java.util.List;

public interface ListItemRepositoryCustom {
    public List<ListItem> findAll(ListItemType listType, boolean irgnoreArchived);

    public List<ListItem> findAllOrderByIndex(ListItemType listType, boolean irgnoreArchived);
}

File: src/main/kotlin/com/patho/main/repository/jpa/custom/ListItemRepositoryCustom.kt
package com.patho.main.repository.jpa.custom

import com.patho.main.model.preset.ListItem
import com.patho.main.model.preset.ListItemType

interface ListItemRepositoryCustom {
    fun findAll(listType: ListItemType, ignoreArchived: Boolean): List<ListItem>

    fun findAllOrderByIndex(listType: ListItemType, ignoreArchived: Boolean): List<ListItem>
}